### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by josex2r**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+			'.huskyrc.js',
+			'.commitlintrc.js',
+			'.eslintrc.js',
+			'.template-lintrc.js',
+			'ember-cli-build.js',
+			'index.js',
+			'testem.js',
+			'blueprints/*/index.js',
+			'config/**/*.js',
+			'lib/**/*.js',
```
Comment (adrigzr) [inline .eslintrc.js]: This folder does not exist.

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+			'node-tests/**/*.js',
+			'tests/dummy/config/**/*.js',
+			'tests/dummy/lib/**/config/**/*.js'
```
Comment (adrigzr) [inline .eslintrc.js]: This folder does not exist.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    - env: EMBER_TRY_SCENARIO=ember-release
+    - env: EMBER_TRY_SCENARIO=ember-beta
+    - env: EMBER_TRY_SCENARIO=ember-canary
+    - env: EMBER_TRY_SCENARIO=ember-default-with-jquery
+    - env: EMBER_TRY_SCENARIO=ember-release
+      node_js: lts/erbium
+    - env: EMBER_TRY_SCENARIO=ember-release
+      node_js: lts/dubnium
+    - env: EMBER_TRY_SCENARIO=ember-release
+      node_js: lts/carbon
```
Comment (adrigzr) [inline .travis.yml]: You are not doing node testing, so this matrix is not necessary.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+# cross-transition-end
+
+[![Build Status](https://travis-ci.com/BBVAEngineering/cross-tansition-shake.svg?branch=master)](https://travis-ci.com/BBVAEngineering/cross-tansition-shake)
+[![GitHub version](https://badge.fury.io/gh/BBVAEngineering%2Fcross-tansition-shake.svg)](https://badge.fury.io/gh/BBVAEngineering%2Fcross-tansition-shake)
+[![NPM version](https://badge.fury.io/js/cross-tansition-shake.svg)](https://badge.fury.io/js/cross-tansition-shake)
+[![Dependency Status](https://david-dm.org/BBVAEngineering/cross-tansition-shake.svg)](https://david-dm.org/BBVAEngineering/cross-tansition-shake)
+[![codecov](https://codecov.io/gh/BBVAEngineering/cross-tansition-shake/branch/master/graph/badge.svg)](https://codecov.io/gh/BBVAEngineering/cross-tansition-shake)
+[![Greenkeeper badge](https://badges.greenkeeper.io/BBVAEngineering/cross-tansition-shake.svg)](https://greenkeeper.io/)
```
Comment (adrigzr) [inline README.md]: Fix the repository name on badges.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+## Information
+
+[![NPM](https://nodei.co/npm/cross-tansition-shake.png?downloads=true&downloadRank=true)](https://nodei.co/npm/cross-tansition-shake/)
+
+Cross-browser transtionend event listener
+
+## Installation
+
+```
+npm install cross-transition-end
```
Comment (adrigzr) [inline README.md]: Repository name on examples.

File: addon/utils/on-transition-end.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+		}
+
+		if (once) {
+			element.removeEventListener(type, fn, true);
+		}
+
+		run(null, callback, e);
+	};
+
+	element.addEventListener(transitionEndEventName, fn, true);
```
Comment (adrigzr) [inline addon/utils/on-transition-end.js:51]: I think it is useful to return `fn` or an unregister function here in order to be able to remove this listener when the consumer wants to.

**Review by adrigzr — CHANGES_REQUESTED** [review]

File: addon/utils/on-transition-end.js (hunk 6 above)
Comment (josex2r) [reply addon/utils/on-transition-end.js:51]: Opened #2

**Review by adrigzr — APPROVED** [review]

**Pull request merged**